vmdk_plugin/main.go

```diff
@@ -120,13 +136,7 @@ func main() {
 		"config":    *configFile,
 	}).Info("Starting plugin ")
 
-	// Load the configuration if one was provided.
-	c, err := config.Load(*configFile)
-	if err != nil {
-		log.Warning("Failed to load config file %s: %v", *configFile, err)
-	}
-
-	// The vmdk driver doesn't depend on the config file for options.
+	// The vSphere driver doesn't depend on the config file for options
 	if *driverName == photonDriver && err == nil {
```

Comment:
It's awkward to chck "err == nil" which is set earlier in the function. I think code can be restructured a bit to do validation in beginning.